coalib/core/Bear.py

```diff
@@ -28,6 +30,15 @@ class Bear:
     will probably want to look at the ``ProjectBear`` and ``FileBear`` classes
     that inherit from this class.
 
+    Attributes for a new bear:
```

Comment:
Maybe you should incorporate a doctest, something like
```
To instantiate a bear, pass a section which contains the bear setting and a file-dictionary (that maps filenames to the according file contents):

>>> section = Section('my-section')
>>> file_dict = {'file.py': ['This is a one-liner\n'']}
>>> Bear(section, file_dict)
```
As when I read "Attributes for a new bear", I wouldn't relate that I have to pass the following values into the constructor.